internal/pkg/agent/application/managed_mode.go

```diff
@@ -139,26 +143,14 @@ func (m *managedConfigManager) Run(ctx context.Context) error {
 		close(retrierRun)
 	}()
 
-	// Gather errors from the dispatcher and pass to the error channel.
-	go func() {
-		for {
-			select {
-			case <-ctx.Done():
-				return
-			case err := <-actionDispatcher.Errors():
-				m.errCh <- err // err is one or more failures from dispatching an action
-			}
-		}
-	}()
-
```

Comment:
The dispatcher's error channel should be emptied by the coordinator as it's returned by the managed mode's `ActionErrors()` method